Sources/XCHammer/XCBuildSettings.swift

```diff
@@ -211,6 +215,7 @@ struct XCBuildSettings: Encodable {
         var container = encoder.container(keyedBy: CodingKeys.self)
 
         try container.encode(copts.joined(separator: " "), forKey: .copts)
+        try container.encode(swiftCopts.joined(separator: " "), forKey: .copts)
```

Comment:
@rahul-malik did you intend to set `OTHER_SWIFT_FLAGS` here? This LOC overwrites `OTHER_CFLAGS`